load/BlimpLanguageService.py

```diff
@@ -145,6 +145,7 @@ def getCopyrightByFilesetId(filesetId):
         copyright = row[0]
         copyrightDate = row[1]
         copyrightDescription = row[2]
-        records.append((copyright, copyrightDate, copyrightDescription))
+        openAccess = row[3]
```

Comment:
I don't think we use openAccess. Remove

Reply:
@bradflood , I have removed this column.